### DISCUSSION TITLE
add ui scaling for osu!mania columns

### DISCUSSION THREAD

**Discussion by Dopaminos** [ppy/osu#21388] [Ideas] [answered]
i wish to change size of o!m columns to make classic o!m skin at least playable in 4k.
i just don't really like messing in skin.ini to find a good size

just look at mwc skin size
![image](https://user-images.githubusercontent.com/42616718/203502239-2da5c1b5-dc5b-4047-8440-4e807723e175.png)

meanwhile classic skin ingame
![image](https://user-images.githubusercontent.com/42616718/203502564-d4b573f7-b860-4b32-9e1c-5b03b5033871.png)

(and add possibility to move skin horizontally as well. not every player likes playing center)

**Comment by peppy** [comment, marked as answer]
We're planning to add skin customisation support for gameplay components in the near future. For now you can use the `skin.ini` specification to make changes like this, but it's a bit more involved (check the osu! wiki for more details).
